manifests/init.pp

```diff
@@ -58,6 +60,7 @@
   Boolean $enable_file = true,
   Boolean $enable_docker = true,
   Boolean $enable_deb = true,
+  Boolean $enable_ansible_collection = true,
```

Comment:
If we go this route (https://github.com/theforeman/puppet-katello/pull/402) this file wont need any changes.